packages/bookings-generator/src/api/api_v1/endpoints/bookings.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from fastapi import APIRouter, Depends, HTTPException
+
+from generator.bookings_generator import get_bookings
+
+
+router = APIRouter()
+
+
+@router.get("/bookings/",
+            summary="Pick the next locations for bookings.",
```

Comment:
this summary is confusing. 
I interpret this route as retrieving a list of bookings